webapp/sources/rudder/rudder-web/src/main/scala/bootstrap/liftweb/RudderUserDetailsFile.scala

```diff
@@ -166,17 +169,19 @@ object UserDetailList {
             Nil
         }
 
-        // Warn that some users  with name different only by their case are defined, and that disabling case sensitivity would break those users
+        // Notice that some users  with name different only by their case are defined, and that disabling case sensitivity would break those users
+        // (not a warning, since we are in the case sensitive case, it's ok)
         ApplicationLogger
-          .warn(s"Users with potential username collision if case sensitivity is disabled: ${res.keys.mkString(", ")}")
+          .info(s"Users with potential username collision if case sensitivity is disabled: ${res.keys.mkString(", ")}")
```

Comment:
we don't warn for things configured by the user to work like that. A warning is something requiring a corrective action by the user (even if not urgent). Warning ends up as ticket created by the log analysis tools of ops team.